### ISSUE TITLE
Admin\Operations - Grid has status column

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1993]
For External Users, change the "registration status" column to "status"
The status options are
Registered
Continue Registration
Transferred
Closed
Temporarily Shutdown

If an operation that was previously closed or temporarily shutdown, and has been reported as Restarted, the status reverts back to Registered. 
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7763-4636&t=Ci7zKMcvo5ipOiRH-4)

For Internal Users, add a "status" column
The status options are the same as the external user, minus the "continue registration"
Registered
Transferred
Closed
Temporarily Shutdown
~~TBD: attn @zoeyli-46 would there be a need for internal users to view incomplete registration data? i.e., should the "Continue Registration" operations be visible to internal users at all?~~

Internal users are able to view incomplete registration data, and are able to see operations which have a "continue registration" status

[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7861-688&t=Ci7zKMcvo5ipOiRH-4)

### Development Details

- we can re-use the `status` column on `Operation` that we had in Registration Part 1, but with modifications. 
- we will continue using the "DRAFT" and "NOT STARTED" status options that we had in Reg Part 1. "NOT STARTED" status applies to operations imported from SWRS where no external user has reviewed the imported data. "DRAFT" status applies to either (1) an operation (either manually created by a user or existing from Part 1) that is part-way through the registration process
- [ ] add options for `Operation.Status` column: "REGISTERED", "TRANSFERRED", "CLOSED", "TEMPORARILY SHUTDOWN"
- in Part 2 we will no longer have a need for the "PENDING", "APPROVED", "DECLINED", or "CHANGES_REQUESTED" statuses that we have in Part 1, but we don't want to delete these statuses at this time because they are in use in production data

#### Development Checklist:

- [x] In `bciers/apps/administration/app/components/datagrid/models/operationColumns.ts` reflect `Status` column similar to `bciers/apps/registration1/app/components/datagrid/models/operationColumns.ts`
- [x] In `bc_obps/registration/models/operation.py` add options for `Operation.Status` column: "REGISTERED", "TRANSFERRED", "CLOSED", "TEMPORARILY SHUTDOWN"
- [x] the operation status of `DRAFT` (as it is named in the database) should be re-labelled as `Continue Registration` on the frontend for external users
- [x] **Sep:** As part of this ticket I have also added the BORO ID column to the Operation data grid
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by zoeyli-46** [comment]
@andrea-williams @hannavovk I added this ticket in to reflect the new status column for internal users and the revised status column for external user. Currently sitting in backlog but I added some labels to it!  I will add backlog refinement label on it in case we want to drag it into an upcoming sprint?

**Comment by hannavovk** [comment]
RE the "TBD: would there be a need for internal users to view incomplete registration data? i.e., should the "Continue Registration" operations be visible to internal users at all?"
- I think there's not need to hide the operations data from internal users even if they have not completed the registration. The incomplete data is effectively the Reg 1 data and it  would also make it easy for them to see which registrations are incomplete and potentially follow up with operators. @zoeyli-46, what do you think? (cc @andrea-williams )

**Comment by zoeyli-46** [comment]
Yes, they will just see the registration 1 data and be able to prompt operators to do their registration. Adding this to the ticket.

**Comment by Sepehr-Sobhani** [comment]
I believe this part: `If an operation that was previously closed or temporarily shutdown, and has been reported as Restarted, the status reverts back to Registered.` Should be handled in ticket: https://github.com/orgs/bcgov/projects/122/views/2?filterQuery=restart&pane=issue&itemId=75557430

**Comment by Sepehr-Sobhani** [comment]
Reachable at:
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/operations

**Comment by patriciarussellCAS** [comment]
Reviewed external view only. Will close this ticket and create any required follow-up ticket for internal views if necessary. 
Design backlog notes for future: 
- Use the same style for Status as shown in Design Guidelines (figma)
- Consider moving the columns in the Operations table so it's Facilities | Status | Action
- Since status and action are related, it's much easier for the users to see/understand the two when they are side by side. Just an idea - happy to be told why it is not a good idea :)
